src/harness/testcases/WINNF_FT_S_SCS_testcase.py

```diff
@@ -64,3 +64,48 @@ def test_WINNF_FT_S_SCS_5(self):
     Checks that a CBSD registration with this configuration succeed.
     """
     self.doTestCipher('ECDHE-RSA-AES128-GCM-SHA256')
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_SCS_6(self):
+    """Unrecognized root of trust certificate presented during registration.
+    Checks that SAS UUT response with fatal alert with unknown_ca.
+    """
+    device_cert = self.getCertFilename('unrecognized_device.cert')
+    device_key = self.getCertFilename('unrecognized_device.key')
+    self.assertTlsHandshakeFailure(device_cert, device_key)
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_SCS_7(self):
+    """Corrupted certificate presented during registration.
+    Checks that SAS UUT response with fatal alert message.
+    """
+    device_cert = self.getCertFilename('corrupted_client.cert')
+    device_key = self.getCertFilename('corrupted_client.key')
+    self.assertTlsHandshakeFailure(device_cert, device_key)
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_SCS_8(self):
+    """Self-signed certificate presented during registration.
+    Checks that SAS UUT response with fatal alert message.
+    """
+    device_cert = self.getCertFilename('self_signed_client.cert')
+    device_key = self.getCertFilename('client.key')
+    self.assertTlsHandshakeFailure(device_cert, device_key)
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_SCS_9(self):
+    """Non-CBRS trust root signed certificate presented during registration.
+    Checks that SAS UUT response with fatal alert message.
+    """
+    device_cert = self.getCertFilename('non_cbrs_signed_device.cert')
+    device_key = self.getCertFilename('non_cbrs_signed_device.key')
+    self.assertTlsHandshakeFailure(device_cert, device_key)
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_SCS_10(self):
```

Comment:
The corresponding test specification seems buggy for me:
the TLS handshake should success, and then the registration request (missing in the code, as reported by Takashi) should failed at the application level...

The current spec expect that the TLS handshake failed, so there is no point to do a registration in this case...

Reply:
After better reading the SCS.10 test specification:
```
Step 2: does a TLS handshake
Check 2: check failure, with information:
If all of the above conditions are met, the SAS UUT PASSES this test. If a TLS session is established,
proceed.
Step 3: does a registration...
```
So if the TLS handshake failed, the test succeed and the test case should end,
otherwise the test should try a registration.

In your  implementation, if the first TLS handshake failed, you expect that the second TLS handshake done during the registration request will succeed...